src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -2268,7 +2302,7 @@
           ]
         },
         {
-          "condition": "(!useHttp)",
+          "condition": "(!useHttp || !useHttpDefault || !useHttpRefit || !useHttpKiota)",
```

Comment:
The default WeatherForecast service stuff uses Refit, so these needs to only be excluded if we are specifically not using HttpRefit